plugins/source/aws/docs/tables/aws_route53_domains.md

```diff
@@ -15,23 +15,24 @@ The composite primary key for this table is (**account_id**, **domain_name**).
 |account_id (PK)|String|
 |domain_name (PK)|String|
 |tags|JSON|
+|admin_contact|JSON|
+|nameservers|JSON|
+|registrant_contact|JSON|
+|tech_contact|JSON|
 |abuse_contact_email|String|
 |abuse_contact_phone|String|
-|admin_contact|JSON|
 |admin_privacy|Bool|
 |auto_renew|Bool|
 |creation_date|Timestamp|
 |dns_sec|String|
 |expiration_date|Timestamp|
-|nameservers|JSON|
-|registrant_contact|JSON|
 |registrant_privacy|Bool|
 |registrar_name|String|
 |registrar_url|String|
 |registry_domain_id|String|
 |reseller|String|
-|status_list|JSON|
-|tech_contact|JSON|
+|status_list|StringArray|
```

Comment:
This looks like a breaking change (changing type JSON to StringArray). Not much we can do about it... but it should technically require a major version bump.